src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -1268,4 +1276,227 @@ public void parseToggleCommand(String[] split) throws TownyException {
 		}
 	}
 
+	public static void handleMetaCommand(Player player, String[] split) {
+		Town town = null;
+		TownyUniverse townyUniverse = TownyUniverse.getInstance();
+
+		try {
+			town = townyUniverse.getDataSource().getResident(player.getName()).getTown();
+		} catch (Exception e) {
+			TownyMessaging.sendErrorMsg(player, e.getClass().toString());
+			return;
+		}
+
+		if (split.length == 2) {
+			if (!player.hasPermission("towny.meta.modify")) {

```

Comment:
Change permission test over to our standard:
if (!townyUniverse.getPermissionSource().testPermission(player, PermissionNodes.TOWNY_COMMAND_TOWNYADMIN_TOWN_META.getNode()))
				throw new TownyException(TownySettings.getLangString("msg_err_command_disable"));